docs/server/source/production-deployment-template/troubleshoot.rst

```diff
@@ -104,3 +104,28 @@ MongoDB instance correctly.
 
 More information about this configuration is provided in
 :doc:`this document <cloud-manager>`.
+
+6. Create a Persistent Volume from existing Azure disk storage Resource
+---------------------------------------------------------------------------
+The configuration file to create PV for Mongo Stateful Set is located at ``mongodb/mongo-pv.yaml``.
+Set the ``name``, ``diskName`` and ``diskURI`` values and run the following command to create PVs
```

Comment:
The order of the instructions here is backwards. First they need to get the diskName and diskURL values, then put them into the `mongodb/mongo-pv.yaml` file. Only then will they do `$ kubectl --context <context-name> apply -f mongodb/mongo-pv.yaml`